packages/firebase_database/firebase_database_web/test/firebase_database_web_test.dart

```diff
@@ -1,3 +0,0 @@
-void main() {
-  // TODO add tests
-}
```

Comment:
is it not required?

Reply:
not much to test on web really, this is mostly a proxy to js objects

Reply:
ok